plugins/churn/src/error/mod.rs

```diff
@@ -0,0 +1,275 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(unused)]
+
+//! An error type suitable for use in Hipcheck's query system.
```

Comment:
Plugins aren't really supposed to be doing internal caching with salsa. We're doing it exclusively with `churn` and `entropy` right now because of concerns about latency making a `linguist` request for every single file in a repo. In the future we'll implement batched queries so we can do away with this `salsa` usage.